site/docs/get_started/for_developers.mdx

```diff
@@ -49,6 +49,31 @@ Import components:
 import { TextInput } from "hds-react";
 ```
 
+**I am not seeing any styles**  
+`hds-react` uses CSS modules under the hood. If your tooling, for instance webpack, is missing support for CSS modules, components may be left without styles altogether. Here's a quick example of how you can declare support for CSS modules without changing behavior for regular CSS files. These rules assume that the files meant for CSS module support have been named like so: `*.module.css`.
```

Comment:
To avoid duplication of documentation I suggest we only reference the instructions in hds-react from here. Unless there is a reason that won't work?

Reply:
Ah, one of them is in the site, another the README. Hmm! Never mind, I think!

Reply:
Still a good point imo! I'd rather have click links and have up to date information than read instructions that have gone stale because they have been forgotten.

But yeah, maybe not the right moment to do it 👍